src/api/app/models/concerns/staging_project.rb

```diff
@@ -227,6 +229,15 @@ def add_managers_group
     assign_managers_group(staging_workflow.managers_group)
   end
 
+  def create_project_log_entry
+    ProjectLogEntry.create!(
+      project: self,
+      user_name: User.current_or_nobody.login, # User.current should be defined, but we don't want this to fail if it's somehow not
```

Comment:
audit logs are pretty useless if they say nobody

Reply:
I agree... I tried it, but it's true that it's useless if they say nobody.